### PR TITLE
Use Cartographer's cmake functions for ROS projects.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

**Comment by fooosmar** [comment]
pull request

**Comment by fooosmar** [comment]
pull request

**Comment by SirVer** [comment]
ptal.

**Comment by SirVer** [comment]
Reassigning to damon since he reviewed all the related code in cartographer.

File: cartographer_ros/cartographer_ros/msg_conversion.cc [hunk 1]
```diff
@@ -34,7 +34,7 @@
 #include "sensor_msgs/MultiEchoLaserScan.h"
 #include "sensor_msgs/PointCloud2.h"
 
-#include "time_conversion.h"
+#include "cartographer_ros/time_conversion.h"
```
Comment (wohe) [inline cartographer_ros/cartographer_ros/msg_conversion.cc]: Move up to the other includes?

File: cartographer_ros/cartographer_ros/sensor_bridge.h [hunk 2]
```diff
@@ -24,13 +24,13 @@
 #include "geometry_msgs/TransformStamped.h"
 #include "nav_msgs/OccupancyGrid.h"
 #include "nav_msgs/Odometry.h"
-#include "sensor_data.h"
+#include "cartographer_ros/sensor_data.h"
```
Comment (wohe) [inline cartographer_ros/cartographer_ros/sensor_bridge.h]: Sort?

**Pull request merged**